Update static links to meetup website

2ndkauboy opened the pull request:
Closes #20

2ndkauboy committed:
update static links to meetup website

Zodiac1978 commented:
LGTM. 

Da ich jetzt über 3 Wochen keine Antwort von @krafit auf Slack bekommen habe (trotz mehrfacher Nachfragen), wie der Status ist, und ob er die API reparieren kann, macht es wahrscheinlich Sinn, das hier zu mergen, um mal die kaputten Links auf den Meetup-Websites zu fixen.

Zodiac1978 commented:
Bin nochmal alle Links durchgegangen und doch noch zwei Probleme gefunden:
https://wpmeetup-karlsruhe.de/ (Fatal Error)
https://wpmeetup-bremen.de/ (Zertifikatsfehler, leitet aber von http auf https)

Zodiac1978 commented:
Habe beide Meetups mal über den Slack-Channel angeschrieben.

Zodiac1978 commented:
Rückmeldung aus Bremen: Erstmal den Meetup-Link nehmen:
https://www.meetup.com/de-DE/bremen-wordpress-meetup-group/

Zodiac1978 committed:
Update URL for Bremen and remove superfluous character

Zodiac1978 committed:
Fix Coding standards issues

PHP CodeBeautifier applied, fixed outdated comments and added missing function visibility.

Zodiac1978 commented:
Karlsruhe läuft wieder

Zodiac1978 reviewed (approved):
LGTM
